### DISCUSSION TITLE
Ossec Config - Reports

### DISCUSSION THREAD

**Discussion by tootsweets** [wazuh/wazuh#17764] [General]
Hi All

I have several reports configured, (examples below) however, I only seem to be getting some of them at 00:00 I,e, only 2 arrive in my inbox

should I receive all reports every morning at 00:00?

is there an issue with how I have implemented the reports? 

```
  <reports>
    <title>Daily report: Successful Authentication Report</title>
    <group>authentication_success</group>
    <email_to>my@email.com</email_to>
    <showlogs>yes</showlogs>
  </reports>
  
  <reports>
    <title>Daily Report: New Packages Installed Via Yum</title>
    <rule>2932</rule>
    <email_to>my@email.com</email_to>
    <showlogs>yes</showlogs>
  </reports>
  
  <reports>
    <title>Daily Report: Failed Authentication Report</title>
    <group>authentication_failed</group>
    <email_to>my@email.com</email_to>
    <showlogs>yes</showlogs>
  </reports>
```

Thanks for your time

**Comment by mdiego92** [comment]
Hello @tootsweets,

It might be possible that you're not receiving the report because there are no alerts to notify.
Could you please verify which report you didn't receive and if you received any alerts of the groups/rule you configured for each `<report>`?

Looking forward to your feedback!

**Comment by tootsweets** [comment]
Hi

Yes I deliberately installed content, logged in via sshd and made some failed attempts for auth to ensure there was data there for the reports 

Thanks

**Reply by mdiego92** [reply]
Hello @tootsweets,

Could you please check your `/var/ossec/logs/ossec.log` file? You should have logs like these for each `<report>` stanza configured.

```
2023/07/07 00:00:10 wazuh-monitord: INFO: Starting daily reporting for 'Daily report: File changes'
2023/07/07 00:00:30 wazuh-monitord: INFO: Starting daily reporting for 'Daily report: alerts 3+'
```

Another thing I would like to check. Do you have the `<alerts_log>yes</alerts_log>` parameter enabled in your Manager's `ossec.conf` file?

Looking forward to your response!
